kotest-framework/kotest-framework-api/src/commonMain/kotlin/io/kotest/core/test/Description.kt

```diff
@@ -101,7 +103,13 @@ sealed class Description {
    /**
     * Returns the name of this description formatted for display.
     */
-   fun displayName() = name.displayName
+   fun displayName(): String {
+      return if (sysprop(KotestEngineSystemProperties.allowMultilineTestName, "FALSE").toUpperCase() == "TRUE") {
+         val spaceCharacter = " "
+         val emptyCharacter = ""
+         name.displayName.split(spaceCharacter).filterNot { it == emptyCharacter }.joinToString(spaceCharacter)
```

Comment:
What about adding a test that this works on \n only. So

"""this
is
my
test"""

Reply:
@sksamuel 
here is no `\n` in this multiline string. 
So something of below screenshot wont work. It does not have \n or " " character so it will appear as `3thirdtestlastline`

<img width="534" alt="Screenshot 2020-10-02 at 10 38 58 PM" src="https://user-images.githubusercontent.com/34642693/94950314-14152100-0500-11eb-8e22-5e7c7e89ae2e.png">

However this will work find as it has " " character in it. It will appear as `3 third test last line` 
<img width="891" alt="Screenshot 2020-10-02 at 10 42 21 PM" src="https://user-images.githubusercontent.com/34642693/94950619-8b4ab500-0500-11eb-8876-b27f6aebcd1c.png">

Which one of the above two case are you referring?

Reply:
I'm referring to the top screenshot. That should still display as "3 third test last line". Instead of splitting on " " you should split on \s

Reply:
Got it thanks @sksamuel

Reply:
Done